### PR TITLE
chopsticks e2e poc

### PR DESCRIPTION
**Pull request by gluneau**
**Pull Request Summary**

> Describe what changes this pull request makes to the repository

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**Release notes:**

- (ex: feat: updated dApp staking to V2)
- (ex: fix: modified styling bugs on the Assets page)

**This pull request makes the following changes:**

**Adds**

- (ex: Add feature A)

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

> \*Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

**Comment by sirius651** [comment]
when I testing this, it works as a charm on my end

File: tests/zombienet/start-chopsticks.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  cp.unref();
+  return cp;
+};
+
+// Execute the run function with the given parameters
+async function executeRun() {
+  try {
+    const childProcess = spawnDetached(
+      'npx @acala-network/chopsticks@latest xcm -p astar -p shiden'
+    );
```
Comment (shaunxw) [inline tests/zombienet/start-chopsticks.js:30]: Another way to interact with Chopsticks is to use the `setupContext` helper provided by `chopsticks-testing` module. It requires less setup works. Here is an example: https://github.com/AstarNetwork/Astar/blob/96b1503a4d7a99141b5c08df1091fbcac14746f4/tests/e2e/tests/runtime-upgrade.test.ts#L17

Comment (gluneau) [reply tests/zombienet/start-chopsticks.js:30]: Very nice, how would you express HRMP channels between parachain this way?

Comment (shaunxw) [reply tests/zombienet/start-chopsticks.js:30]: You can setup multiple `setupContext` instances. There are some examples of XCM testing in Acala's e2e test repo.

**Comment by sirius651** [comment]
Thanks. This [PR](https://github.com/AstarNetwork/astar-apps/pull/789) includes all of logic here. so maybe can we review that instead? @gluneau

**Pull request closed without merging**